config/config.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/viper"
+)
+
+type config struct {
+	CSV struct {
+		Path string
+	}
+	Server struct {
+		Address string
+	}
+	Logging bool
+}
+
+var C config
```

Comment:
By convention, all the exported stuff (variables, functions, etc) should have a description
```go 
// C - Description here
```

Please do this for all the exported stuff

Reply:
Are you OK with me adding those in the second deliverable branch? 
With the amount of changes in that other branch would mean adding a lot of comments for things that are going to disappear

Reply:
See the comments now in the SECOND-DELIVERABLE PR [here](https://github.com/hamg26/academy-go-q42021/pull/2/commits/94c4f3df1986e94dfa4774892c63189909479933)